docs/io/optional/logging_configuration.ipynb

```diff
@@ -0,0 +1,2226 @@
+{
```

Comment:
<p>Remove the i.e. section so it reads "the <code>config["debug"]["logging_level"]</code> is set to <code>Debug</code> via the <strong>YAML</strong> file."</p><p>Reword second sentence to be "The user will also set the log_state parameter as Info."</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/tardis-sn/tardis/pull/1633/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='47'/>